lib/puppet/transaction/report.rb

```diff
@@ -256,8 +256,8 @@ def initialize_from_hash(data)
     @time = data['time']
     @corrective_change = data['corrective_change']
 
-    if data['master_used']
-      @master_used = data['master_used']
+    if data['server_used']
+      @server_used = data['server_used']
     end
```

Comment:
To preserve backwards compatibility when deserializing an older report, I think we want to check for `server_used`, but it if's not set, then fallback to `master_used`.